pkg/cli/environment.go

```diff
@@ -26,23 +26,23 @@ import (
 	"fmt"
 	"os"
 	"strconv"
+	"sync"
 
 	"github.com/spf13/pflag"
 
 	"helm.sh/helm/v3/pkg/helmpath"
+	"helm.sh/helm/v3/pkg/kube"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
```

Comment:
here's the line failing CI. This should be moved up beside the pflag import.

General practice for Go import statements is

```
import (
    // standard libraries

    // third-party packages

    // local packages
)
```

Reply:
Yeah, It took me a bit to figure that out. Running make format fixed it up for me.